packages/blade/jest.web.config.js

```diff
@@ -0,0 +1,21 @@
+const ignores = ['/node_modules/'];
+
+module.exports = {
+  testPathIgnorePatterns: [...ignores, 'native.test'],
+  collectCoverageFrom: ['./src/**/*.web.{ts,tsx}'],
+  coverageThreshold: {
+    global: {
+      branches: 75,
+      functions: 75,
+      lines: 75,
+      statements: 75,
+    },
+  },
+  moduleFileExtensions: ['web.ts', 'web.tsx', 'ts', 'tsx', 'js', 'json', 'node'],
+  testMatch: ['**/__tests__/**/*.web.test.{ts,tsx}', '**/__tests__/**/*.test.{ts,tsx}'],
```

Comment:
testing all the files that matches
`.web.test.ts|js|tsx|jsx` and platform independednt files `.test.ts|js|tsx|jsx`